docroot/modules/custom/sitenow_people/sitenow_people.module

```diff
@@ -643,6 +698,35 @@ function sitenow_people_preprocess_views_view(&$variables) {
   }
 }
 
+/**
+ * Helper function to preprocess rooms list.
+ */
+function sitenow_people_preprocess_people_list(&$variables, ...$fields_to_hide) {
```

Comment:
Since the main purpose of this function seems to be hiding fields (specifically in views results because of `$variables['rows']`), I think we should rename the function to be more specific to what it does. Also the term "preprocess" might be a bit loaded in the context of Drupal.

Reply:
```suggestion
function sitenow_people_views_row_hide_fields(&$variables, ...$fields_to_hide) {
```